src/pages/virtual-machines/virtual-machines-list/VirtualMachinesList.tsx

```diff
@@ -194,7 +193,14 @@ const VirtualMachinesList = () => {
           field="vmBackupPolicy"
           header="Policy"
           sortable
-          body={(rowData) => rowData?.vmBackupPolicy?.name}
+          body={(rowData) => {
```

Comment:
body={({vmBackupPolicy}) => vmBackupPolicy && vmBackupPolicy.name.startsWith('uuid') ? vmBackupPolicy.name.split.... : vmBackupPolicy.name}

Reply:
destructuring makes an error due to not existing **{vmBackupPolicy}**